src/utils/googleAnalytics.ts

```diff
@@ -57,6 +46,7 @@ export const loadGoogleAnalytics = (): void => {
     anonymizeIp: true,
     appName: `Gnosis Safe Web`,
     appVersion: process.env.REACT_APP_APP_VERSION,
+    dimension1: getNetworkInfo().label, // Chain
```

Comment:
So every event has the dimension, and also init? It's a bit weird.

Reply:
I removed it from the init.